src/components/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { TextOrientation, Type } from './Divider.types'
+import { Divider } from '.'
+
+const meta = {
+  args: {
+    text: 'Customize me!',
+  },
+  argTypes: {
+    text: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+  component: Divider,
```

Comment:
```suggestion
  component: Divider,
  args: Divider.defaultProps,
```

Reply:
That should be enough (`argTypes` is not needed because the prop is already declared as a string)

Reply:
without declaring `text` type the story is showing the field as json actually, this has been added to only allow text input